open-models/serving/vertex_ai_text_generation_inference_gemma.ipynb

```diff
@@ -0,0 +1,794 @@
+{
```

Comment:
<p>Can you move this section of <code>Requirements</code> section  after the <code>Authenticate your Hugging Face account</code> section? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/1064/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
P.S. I've placed that under the Get Started section, but let me know if you meant otherwise!